app/hooks/use-mods-directory-watcher.ts

```diff
@@ -15,11 +19,16 @@ export function useModsDirectoryWatcher(owmlPath: string, handler: Handler) {
 
     debugConsole.log('useEffect: useModsDirectoryWatcher');
     const path = `${owmlPath}/Mods`;
+    const patha = `${alphaPath}/BepInEx/plugins`;
```

Comment:
First of all, bad naming all around this file. But you're repeating a lot of unnecessary code anyway. Just fetch the `isAlphaMode` value from the settings store (in fact, you can probably fetch owmlPath and alphaPath from the store too instead of passing them as arguments). Then do 
```ts
const path = isAlphaMode ? `${alphaPath}/BepInEx/plugins` : `${owmlPath}/Mods`
```
The rest of the file can be as it was before.